bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/update/AbstractToolsHandler.java

```diff
@@ -120,11 +123,10 @@ protected String getPythonExecutablePath()
 			{
 				Logger.log("No Python installations found in the system."); //$NON-NLS-1$
 			}
```

Comment:
Can we return here itself if there are no python versions found? https://github.com/espressif/idf-eclipse-plugin/blob/2416d02286b3fea2022547f41aa06fb0387fcc45/bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/update/AbstractToolsHandler.java#L124

Reply:
Hi @kolipakakondal, but if pythonVersions are empty, we are trying to get the python executable from IDFUtil::getPythonExecutable. Python can be installed on the windows machine without adding it to the windows registry, in this case, we will always return a null here, even if a user provided the python executable during tools installation.

Reply:
Ok, got it.